luigi/scheduler.py

```diff
@@ -90,6 +90,8 @@ class scheduler(Config):
                                             config_path=dict(section='scheduler', name='disable-window-seconds'))
     disable_failures = parameter.IntParameter(default=None,
                                               config_path=dict(section='scheduler', name='disable-num-failures'))
+    disable_hard_timeout = parameter.TimeDeltaParameter(default=None,
+                                              config_path=dict(section='scheduler', name='disable-hard-timeout'))
```

Comment:
@erikbern, not related to this review. But why are these setting the `config_path`. Isn't it automatic to be `dict(section=$className, name=$fieldName)`?